### PR TITLE
use padding and marging for requirements as well

### PR DESCRIPTION
**Pull request by vhvb1989**
fixes: #1462 

Setting the style of a requirement to do the same as headers (padding for the banner)

Before the fix: https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-service-return-model-public-getters

After the fix: https://vhvb1989.github.io/azure-sdk/dotnet_introduction.html#dotnet-service-return-model-public-getters

### CODE REVIEW

**Review by salameer — APPROVED** [review]

**Comment by vhvb1989** [comment]
@weshaggard  @jongio  please review and approve whenever possible for you.
Thank you

**Comment by annelo-msft** [comment]
This looks right on a first load of the site.  I noticed I get some strange behavior if I reload....

First load:

image

1st reload:

image

2nd reload:

image

**Comment by vhvb1989** [comment]
> I noticed I get some strange behavior if I reload

My bad @annelo-msft , I can only have one Demo at the time and I loaded another version with a different fix.  I have fixed it now and you can try again the link: https://vhvb1989.github.io/azure-sdk/dotnet_introduction.html#dotnet-service-return-model-public-getters

**Comment by annelo-msft** [comment]
That looks great the first time I load it -- thanks!

image

I'm still seeing the behavior where if I click the reload button, I get a strange result:

1st reload:
image

2nd reload:
image

3rd reload:
image

**Comment by vhvb1989** [comment]
> I'm still seeing the behavior where if I click the reload button, I get a strange result:

It looks like a general issue for all anchors in the design page.  I will create an issue to investigate that one.  I suspect there might be some js script running on page load() which makes page offset.

See for example this first anchor: https://azure.github.io/azure-sdk/dotnet_implementation.html#dotnet-usage-httppipeline
We can keep reloading the page and it keeps somehow letting the expected title insider the screen.

Then look at: https://azure.github.io/azure-sdk/dotnet_implementation.html#json-serialization
Only the first time works, then every reload adds more and more offset to the title.

**Comment by vhvb1989** [comment]
Issue created for the reloading page with anchor:
https://github.com/Azure/azure-sdk/issues/2445

**Review by annelo-msft — APPROVED** [review]

File: _layouts/default.html [hunk 1]
```diff
@@ -60,43 +61,58 @@
...
+      -->
+    <script>
+        $(window).bind("load", function () {
+            let requeriments = $('.requirement');
+            requeriments.each(function (index, element) {
+                console.log(index);
+                $(element).css("z-index", requeriments.length - index);
+            });
+        });
+    </script>
```
Comment (vhvb1989) [inline _layouts/default.html:87]: This is the main addition to this file. Do not pay attention to the formatting-only from else where.

**Pull request merged**